uart-protocol.md

```diff
@@ -617,18 +618,18 @@ Example, LEGO BOOST Color and Distance sensor:
 #define   MODE_7                        0x07    // MODE 7 (or 15 if INFO_MODE_PLUS_8 bit is set)
 
 // CMD_EXT_MODE payload
-# define EXT_MODE_0                     0x00    // mode is < 8
-# define EXT_MODE_8                     0x08    // mode is >= 8
+#define   EXT_MODE_0                    0x00    // mode is < 8
+#define   EXT_MODE_8                    0x08    // mode is >= 8
 
 // SECOND INFO BYTE
 
 #define   INFO_NAME                     0x00    // INFO command - NAME    (device name)
 #define   INFO_RAW                      0x01    // INFO command - RAW     (device RAW value span)
 #define   INFO_PCT                      0x02    // INFO command - PCT     (device PCT value span)
 #define   INFO_SI                       0x03    // INFO command - SI      (device SI  value span)
-#define   INFO_SYMBOL                   0x04    // INFO command - SYMBOL  (device SI  unit symbol)
+#define   INFO_UNITS                    0x04    // INFO command - SYMBOL  (device SI  unit symbol)
```

Comment:
should we change `SYMBOL` in the comment too?

Reply:
Indeed comments/texts are now synced (units/symbol & combos/combo).